docs/variants/novacustom_ns5x_adl/hardware-matrix.md

```diff
@@ -78,11 +80,13 @@ for the NS51/70 platforms.
 | **Wireless card**              | Intel Wi-Fi 6 AX201                              |
 | **Display**                    | Display 1: HDMI 1920x1080p                       |
 | **Network**                    | Local network wired connection                   |
-| **Internal devices**           | 1. 1920x1080 14 inch screen                      |
+| **Internal devices**           | 1. 1920x1080 17 inch screen                      |
 |                                | 2. Internal keyboard with LED backlight          |
 |                                | 3. Touchpad                                      |
 |                                | 4. Camera                                        |
 |                                | 5. Audio subsystem                               |
 | **Attached devices**           | Thunderbolt 4 Port: Wavlink 100W PD Pro docking station |
```

Comment:
Since the docks are not permanently connected to any one laptop, I think we should probably remote this note here. Every laptop is tested with every dock / hub listed here: https://github.com/Dasharo/docs/pull/715 which obesoletes the dock sections here I think.

Maybe replace this with the link to the doc containing the dock HCL?

Reply:
You're right, I'll wait for your PR to docs to update it.

Reply:
Done: 734da51b6b6304d1830d9581a9927a0d322dc275

I tried to link to the platform version via the # value, but it doesn't work in our docs system.
I tried to do it like in the examples:
https://squidfunk.github.io/mkdocs-material/reference/grids/#using-generic-grids-ordered-list
https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#embedded-content-ordered-list

Reply:
I found this, but it's in experimental version 9.5.0.
https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#anchor-links

Our server seems to have an older version. On my computer I have:
```
$ pip show mkdocs-material
Name: mkdocs-material
Version: 9.2.6
```

Reply:
Feel free to bump the version to v9.5.0 in mkdocs.yml

Reply:
Sorry, the file is `requirements.txt`. You just have to bump the number there, CI will regenerate the venv and install all dependencies automatically.